### DISCUSSION TITLE
Can you provide an example for incremental sync?

### DISCUSSION THREAD

**Discussion by MP3GMike** [osalvador/ReplicaDB#56] [Q&A] [answered]
If we want to sync once a day, how do we achieve that without repeating data?
and if we wan't to sync multiple tables, do we have to do it table by table?

**Comment by osalvador** [comment, marked as answer]
Hi @MP3GMike ,

- "_If we want to sync once a day, how do we achieve that without repeating data?_" 
 There are many strategies to achieve incremental replication. ReplicaDB has incremental mode, but you need to filter the new data created or updated in the source table (commonly based on incremental IDs or row update date). You can refer to the documentation https://osalvador.github.io/ReplicaDB/docs/docs.html#incremental

- "_ if we wan't to sync multiple tables, do we have to do it table by table?_"
Currently ReplicaDB does not support replication of multiple tables defined in a configuration file. You can use a base configuration file and use environment variables, here's an example https://github.com/osalvador/ReplicaDB/issues/27#issuecomment-862229297

Regards!
